### ISSUE TITLE
Queue time extremely high

### ISSUE THREAD

**Issue by letsgoti** [ValveSoftware/Dota2-Gameplay#9415]
### Description

I have over 5 thousand matches in dota, I quited playing on january. Came back yesterday, played 3 games with a reasonable queue time. Now, I'm trying to play my 4th match, and have been queueing for over 45 minutes. 

Im trying to play in Brazil, Peru and Chile servers. Im calibrating but playing around Legend 5 mmr, as midlaner and carry. 

what the hell is happening it is 4pm now, shouldn't have a long queue time at all.

### Screenshots

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/134645669/abdff7d0-6d42-403e-bac1-347d27de0ba9)

**Comment by Psy7en** [comment]
Im having the same issue although Im queuing for almost 20 mins everytime. Steam needs to fix this asap.

**Comment by letsgoti** [comment]
Yep. and it varies a lot, just found a match within 2 minutes of queue time. Now searching for a second, already 30 min queue time. Gotta love Valve. And reporting this issue here, does it even help with anything?
